examples/MotorImagery/two_class_motor_imagery.py

```diff
@@ -6,34 +6,32 @@
 from sklearn.pipeline import make_pipeline
 
 from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+from sklearn.svm import SVC
 import pandas as pd
 import seaborn as sns
 import matplotlib.pyplot as plt
 
 from collections import OrderedDict
 from moabb.datasets.bnci import (BNCI2014001, BNCI2014002,
-                                 BNCI2014004, BNCI2015001)
+                                 BNCI2014004, BNCI2015001, BNCI2015004)
 
 from moabb.datasets.alex_mi import AlexMI
-from moabb.datasets.bbci_eeg_fnirs import BBCIEEGfNIRS
+# from moabb.datasets.bbci_eeg_fnirs import BBCIEEGfNIRS
 from moabb.datasets.gigadb import GigaDbMI
 from moabb.datasets.physionet_mi import PhysionetMI
 from moabb.datasets.openvibe_mi import OpenvibeMI
+pi = PhysionetMI()
+pi.subject_list = range(1,5)
```

Comment:
should we keep the full dataset ? 
if you want to leave this, please add a comment so that users understand why we added this hack

Reply:
ah -- that's just because using all of them takes forever to download (especially so with the GigaDB stuff). I should move my test script to something that's not version controlled I guess...

Reply:
that's fine, any example script is valuable. I do understand what you did here (I do it too).

I think we should make another example that will actually download and run the moabb on all dataset / subjects.